File: sample/java/com/example/java/LonelyCase.java
package com.example.java;

public class LonelyCase {
    public LonelyCase(int foo, Runnable bar) {
        switch (foo) {
            case 0: bar.run();

            case 2: bar.run();

            default:
                bar.run();
                break;
        }
    }
}

File: sample/c/lonely_case.c
#include <stdio.h>

static int lonely_case() {
    constexpr int expression = 0;
    if (expression == 12) {
        constexpr int foo = 0;
        printf("%d", foo);
    }
    return 0;
}

int main() {
    lonely_case();
    return 0;
}
